api/graphql/schema.graphql

```diff
@@ -912,7 +912,7 @@ type Query {
     @all(model: "Pool", scopes: ["wasPublished"])
     @can(ability: "viewAnyPublishedAdvertisement", model: "Pool")
   poolByKey(key: String! @eq): Pool @find @can(ability: "view", query: true)
-  pools: [Pool]! @all @can(ability: "viewAny")
+  pools: [Pool]! @all(scopes: ["authorizedToView"]) @can(ability: "view", resolved: true)
```

Comment:
The authorizedToView scope is unnecessary here, since all the pools attached to a search request should already by published.

Reply:
Sorry, was this commented in the correct line? This seems to be the general `Query.pools` query which could cause errors if called without the `platform_admin` role (which is the case for `request_responder` currently).

Reply:
Oh! Yes, that's a mistake on my part, I was thinking this line was part of the PoolCandidateSearchRequest block, not the Query block 🤦‍♂️

Reply:
Where are request_responders running the pools query?

Reply:
On the Talent Requests page there is the "Candidate Results" table that queries pools right now

Reply:
Ah, of course